BuildLoader.py

```diff
@@ -446,6 +446,19 @@ def update_fit_table (self):
             print ('  Patching entry %d with 0x%08X:0x%08X - BIOS Module(Stage1B)' % (num_fit_entries, fit_entry.address, fit_entry.size))
             num_fit_entries     += 1
 
+            # TXT POLICY
+            if self._board.TXT_ENABLED == 1:
+                IndexPort = 0x0070 #CMOS index port
+                DataPort  = 0x0071 #CMOS data port
```

Comment:
Where they are used? SBL is designed to be CMOS free, please check if we could make some changes to avoid using CMOS.